src/egon/data/datasets.yml

```diff
@@ -1092,6 +1092,9 @@ emobility_mit:
     export_results_to_csv: True
     parallel_tasks: 10
 
+demand_timeseries_mvgd:
+  parallel_tasks: 10
```

Comment:
We experienced more stable runs with only 5 tasks - change for now until #1034 is solved @birgits ?

Reply:
Sure, I can change it!